### PR TITLE
Add `gauge` alternate icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by jguddas — COMMENTED** [review]
What about calling it circle gouge?

**Review by ericfennis — APPROVED** [review]

File: icons/gauge-circle.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "meter",
+    "speed",
+    "pressure",
+    "measure",
+    "level"
+  ],
+  "categories": [
+    "account",
+    "transportation",
+    "sports",
```
Comment (ericfennis) [inline icons/gauge-circle.json:15]: why is this part of sports?

Comment (ericfennis) [reply icons/gauge-circle.json:15]: Oh I see was already part of `gauge` check.

Comment (danielbayley) [reply icons/gauge-circle.json:15]: > why is this part of sports?

> Oh I see was already part of `gauge` check.

@ericfennis Ah yeah, and in answer to the question, specifically, motorsports…

**Pull request merged**